### PR TITLE
 Bump containers/image to v3.0.2 to fix keyring issue 

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @giuseppe @vrothberg @nalind @ashley-cui @QiWang19 PTAL

**Comment by TomSweeneyRedHat** [comment]
If the dust settles in https://github.com/containers/libpod/pull/3752 on the directory location discussion, then LGTM

File: pkg/cli/common.go [hunk 1]
```diff
@@ -261,5 +261,10 @@ func GetDefaultAuthFile() string {
 	if runtimeDir != "" {
 		return filepath.Join(runtimeDir, "containers/auth.json")
 	}
-	return ""
+
+	uid := os.Getuid()
+	if uid == 0 {
+		return fmt.Sprintf("/run/containers/%d/auth.json", os.Getuid())
+	}
+	return fmt.Sprintf("/run/user/%d/containers/auth.json", os.Getuid())
```
Comment (TomSweeneyRedHat) [inline pkg/cli/common.go]: This differs from https://github.com/containers/libpod/pull/3752 and I'm not sure it's quite right as I think root would hit 267 when it should hit 269.

Comment (mtrmac) [reply pkg/cli/common.go]: The actual behavior depends on whether `XDG_RUNTIME_DIR` is set, and su/sudo clears it. It is not strictly specific to UID=0.

**Comment by rhatdan** [comment]
@mtrmac @TomSweeneyRedHat @QiWang19 @nalind @vrothberg PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM
Now starting a happy ci dance....

**Pull request merged**